test_scripts/Defects/6_1/3139_01_Stop_streaming_gracefully_for_1st_app_if_2nd_is_activated_and_starts_streaming.lua

```diff
@@ -0,0 +1,137 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3139
+--
+-- Steps:
+-- 1. Set StopStreamingTimeout = 3000 in SDL .INI file
+-- 2. Start SDL, HMI, connect Mobile device
+-- 3. Register PROJECTION (App_1) and NAVIGATION (App_2) applications
+-- 4. Activate App_1 and start Video streaming
+-- 5. Deactivate App_1 (streaming still continue since app has STREAMABLE state)
+-- 6. Activate App_2
+-- SDL does:
+--   - send OnHMIStatus(FULL, STREAMABLE) notification to App_2
+--   - send OnHMIStatus(LIMITED, NOT_STREAMABLE) notification to App_1
+--   - send Navi.OnVideoDataStreaming(false) notification to HMI
+--   - send Navi.StopStream(App_1) request to HMI
+--   - send EndService(VIDEO) request to App_1
+--   - start 'StopStreamingTimeout' timeout
+-- 7. App_1 stops streaming within 'StopStreamingTimeout' timeout
+-- SDL does:
+--   - not send OnAppInterfaceUnregistered notification to App_1
+--   - not send BC.OnAppUnregistered notification to HMI
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("test_scripts/Defects/6_1/common_3139_3140_3142")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Apps Configuration ]]
+common.app.getParams(1).appHMIType = { "PROJECTION" } -- NAVIGATION
```

Comment:
Why do we need the comment `-- NAVIGATION` here?
Comment also is related to other scripts.

Reply:
@GetmanetsIrina Please check updates in a755982